pynopegl-utils/pynopegl_utils/tests/cmp.py

```diff
@@ -73,8 +73,8 @@ class CompareSceneBase(CompareBase):
     def __init__(
         self,
         scene_func: Callable[..., ngl.SceneInfo],
-        width: int = 1280,
-        height: int = 800,
+        width: int,
+        height: int,
```

Comment:
Can you add in the commit description that is because of the inheritance of `_CompareFingerprint` ?

Reply:
Added